src/config-parsing/index.test.ts

```diff
@@ -48,7 +49,7 @@ describe(interpolateSecretsIntoConfig.name, () => {
     }).toThrow('Secret cannot be blank');
   });
 
-  it('can use "\\" to escape interpolation', () => {
+  it(String.raw`can use "\" to escape interpolation`, () => {
```

Comment:
What is this rule? I don't really like it.

Reply:
It's new from unicorn: `unicorn/prefer-string-raw`. I'll turn it off along with `@jest/globals`

Reply:
Removed